iOS Command PhaseScriptExecution failed with a nonzero exit code

<!---
**Note <= v5 is deprecated, v5 issues are unlikely to get attention. Feel free to ask in discussions instead.**

Hello there you awesome person;

Please note that the issue list of this repo is exclusively for bug reports;

1) For feature requests, questions and general support please use [GitHub Discussions](https://github.com/invertase/react-native-firebase/discussions).
2) If this is a setup issue then please make sure you've correctly followed the setup guides, most setup issues such as 'duplicate dex files', 'default app has not been initialized' etc are all down to an incorrect setup as the guides haven't been correctly followed.
-->

<!-- NOTE: You can change any of the `[ ]` to `[x]` to mark an option(s) as selected -->

<!-- PLEASE DO NOT REMOVE ANY SECTIONS FROM THIS ISSUE TEMPLATE   -->
<!--   Leave them as they are even if they're irrelevant to your issue -->

## Issue

<!-- Please describe your issue here --^ and provide as much detail as you can. -->
<!-- Include code snippets that show your usages of the library in the context of your project. -->
<!-- Snippets that also show how and where the library is imported in JS are useful to debug issues relating to importing or methods not found issues -->

Describe your issue here
I am aware that this issue has been brought up before and I am very sure that I followed the documentation properly and I have also looked up on those issues that are related to my question.

So I installed the packages `@react-native-firebase/app` and `@react-native-firebase/messaging` because all I want to use Firebase is for Cloud Messaging. I followed the setup as indicated in the documentation and ran the usual `pod install` in the ios folder of my project.  I also added an empty `firebase.json` file that was instructed [here](https://rnfirebase.io/faqs-and-tips#my-ci-build-hangs-at-the-running-script-cp-user-core-configuration-step).

However, after doing all that, my build still fails. With the output saying:
```
The following build commands failed:
PhaseScriptExecution [CP-User]\ [RNFB]\ Core\ Configuration /Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Script-3A3BA8C7294B359F516CA4D5.sh (in target 'Gibber' from project 'Gibber')
``` 

When I open the script that was stated in the output, it shows this:
```
#!/bin/sh
#!/usr/bin/env bash
#
# Copyright (c) 2016-present Invertase Limited & Contributors
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this library except in compliance with the License.
# You may obtain a copy of the License at
#
#   http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
set -e

_MAX_LOOKUPS=2;
_SEARCH_RESULT=''
_RN_ROOT_EXISTS=''
_CURRENT_LOOKUPS=1
_JSON_ROOT="'react-native'"
_JSON_FILE_NAME='firebase.json'
_JSON_OUTPUT_BASE64='e30=' # { }
_CURRENT_SEARCH_DIR=${PROJECT_DIR}
_PLIST_BUDDY=/usr/libexec/PlistBuddy
_TARGET_PLIST="${BUILT_PRODUCTS_DIR}/${INFOPLIST_PATH}"
_DSYM_PLIST="${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Info.plist"

# plist arrays
_PLIST_ENTRY_KEYS=()
_PLIST_ENTRY_TYPES=()
_PLIST_ENTRY_VALUES=()

function setPlistValue {
  echo "info:      setting plist entry '$1' of type '$2' in file '$4'"
  ${_PLIST_BUDDY} -c "Add :$1 $2 '$3'" $4 || echo "info:      '$1' already exists"
}

function getFirebaseJsonKeyValue () {
  if [[ ${_RN_ROOT_EXISTS} ]]; then
    ruby -e "require 'rubygems';require 'json'; output=JSON.parse('$1'); puts output[$_JSON_ROOT]['$2']"
  else
    echo ""
  fi;
}

function jsonBoolToYesNo () {
  if [[ $1 == "false" ]]; then
    echo "NO"
  elif [[ $1 == "true" ]]; then
    echo "YES"
  else echo "NO"
  fi
}

echo "info: -> RNFB build script started"
echo "info: 1) Locating ${_JSON_FILE_NAME} file:"

if [[ -z ${_CURRENT_SEARCH_DIR} ]]; then
  _CURRENT_SEARCH_DIR=$(pwd)
fi;

while true; do
  _CURRENT_SEARCH_DIR=$(dirname "$_CURRENT_SEARCH_DIR")
  if [[ "$_CURRENT_SEARCH_DIR" == "/" ]] || [[ ${_CURRENT_LOOKUPS} -gt ${_MAX_LOOKUPS} ]]; then break; fi;
  echo "info:      ($_CURRENT_LOOKUPS of $_MAX_LOOKUPS) Searching in '$_CURRENT_SEARCH_DIR' for a ${_JSON_FILE_NAME} file."
  _SEARCH_RESULT=$(find "$_CURRENT_SEARCH_DIR" -maxdepth 2 -name ${_JSON_FILE_NAME} -print | head -n 1)
  if [[ ${_SEARCH_RESULT} ]]; then
    echo "info:      ${_JSON_FILE_NAME} found at $_SEARCH_RESULT"
    break;
  fi;
  _CURRENT_LOOKUPS=$((_CURRENT_LOOKUPS+1))
done

if [[ ${_SEARCH_RESULT} ]]; then
  _JSON_OUTPUT_RAW=$(cat "${_SEARCH_RESULT}")
  _RN_ROOT_EXISTS=$(ruby -e "require 'rubygems';require 'json'; output=JSON.parse('$_JSON_OUTPUT_RAW'); puts output[$_JSON_ROOT]" || echo '')

  if [[ ${_RN_ROOT_EXISTS} ]]; then
    _JSON_OUTPUT_BASE64=$(python -c 'import json,sys,base64;print(base64.b64encode(json.dumps(json.loads(open('"'${_SEARCH_RESULT}'"').read())['${_JSON_ROOT}'])))' || echo "e30=")
  fi

  _PLIST_ENTRY_KEYS+=("firebase_json_raw")
  _PLIST_ENTRY_TYPES+=("string")
  _PLIST_ENTRY_VALUES+=("$_JSON_OUTPUT_BASE64")

  # config.app_data_collection_default_enabled
  _APP_DATA_COLLECTION_ENABLED=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "app_data_collection_default_enabled")
  if [[ $_APP_DATA_COLLECTION_ENABLED ]]; then
    _PLIST_ENTRY_KEYS+=("FirebaseDataCollectionDefaultEnabled")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("$(jsonBoolToYesNo "$_APP_DATA_COLLECTION_ENABLED")")
  fi

  # config.analytics_auto_collection_enabled
  _ANALYTICS_AUTO_COLLECTION=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "analytics_auto_collection_enabled")
  if [[ $_ANALYTICS_AUTO_COLLECTION ]]; then
    _PLIST_ENTRY_KEYS+=("FIREBASE_ANALYTICS_COLLECTION_ENABLED")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("$(jsonBoolToYesNo "$_ANALYTICS_AUTO_COLLECTION")")
  fi

  # config.analytics_collection_deactivated
  _ANALYTICS_DEACTIVATED=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "analytics_collection_deactivated")
  if [[ $_ANALYTICS_DEACTIVATED ]]; then
    _PLIST_ENTRY_KEYS+=("FIREBASE_ANALYTICS_COLLECTION_DEACTIVATED")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("$(jsonBoolToYesNo "$_ANALYTICS_DEACTIVATED")")
  fi

  # config.analytics_idfv_collection_enabled
  _ANALYTICS_IDFV_COLLECTION=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "analytics_idfv_collection_enabled")
  if [[ $_ANALYTICS_IDFV_COLLECTION ]]; then
    _PLIST_ENTRY_KEYS+=("GOOGLE_ANALYTICS_IDFV_COLLECTION_ENABLED")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("$(jsonBoolToYesNo "$_ANALYTICS_IDFV_COLLECTION")")
  fi

  # config.analytics_default_allow_ad_personalization_signals
  _ANALYTICS_PERSONALIZATION=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "analytics_default_allow_ad_personalization_signals")
  if [[ $_ANALYTICS_PERSONALIZATION ]]; then
    _PLIST_ENTRY_KEYS+=("GOOGLE_ANALYTICS_DEFAULT_ALLOW_AD_PERSONALIZATION_SIGNALS")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("$(jsonBoolToYesNo "$_ANALYTICS_PERSONALIZATION")")
  fi

  # config.perf_auto_collection_enabled
  _PERF_AUTO_COLLECTION=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "perf_auto_collection_enabled")
  if [[ $_PERF_AUTO_COLLECTION ]]; then
    _PLIST_ENTRY_KEYS+=("firebase_performance_collection_enabled")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("$(jsonBoolToYesNo "$_PERF_AUTO_COLLECTION")")
  fi

  # config.perf_collection_deactivated
  _PERF_DEACTIVATED=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "perf_collection_deactivated")
  if [[ $_PERF_DEACTIVATED ]]; then
    _PLIST_ENTRY_KEYS+=("firebase_performance_collection_deactivated")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("$(jsonBoolToYesNo "$_PERF_DEACTIVATED")")
  fi

  # config.messaging_auto_init_enabled
  _MESSAGING_AUTO_INIT=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "messaging_auto_init_enabled")
  if [[ $_MESSAGING_AUTO_INIT ]]; then
    _PLIST_ENTRY_KEYS+=("FirebaseMessagingAutoInitEnabled")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("$(jsonBoolToYesNo "$_MESSAGING_AUTO_INIT")")
  fi

  # config.in_app_messaging_auto_colllection_enabled
  _FIAM_AUTO_INIT=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "in_app_messaging_auto_collection_enabled")
  if [[ $_FIAM_AUTO_INIT ]]; then
    _PLIST_ENTRY_KEYS+=("FirebaseInAppMessagingAutomaticDataCollectionEnabled")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("$(jsonBoolToYesNo "$_FIAM_AUTO_INIT")")
  fi

  # config.app_check_token_auto_refresh
  _APP_CHECK_TOKEN_AUTO_REFRESH=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "app_check_token_auto_refresh")
  if [[ $_APP_CHECK_TOKEN_AUTO_REFRESH ]]; then
    _PLIST_ENTRY_KEYS+=("FirebaseAppCheckTokenAutoRefreshEnabled")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("$(jsonBoolToYesNo "$_APP_CHECK_TOKEN_AUTO_REFRESH")")
  fi

  # config.crashlytics_disable_auto_disabler - undocumented for now - mainly for debugging, document if becomes useful
  _CRASHLYTICS_AUTO_DISABLE_ENABLED=$(getFirebaseJsonKeyValue "$_JSON_OUTPUT_RAW" "crashlytics_disable_auto_disabler")
  if [[ $_CRASHLYTICS_AUTO_DISABLE_ENABLED == "true" ]]; then
    echo "Disabled Crashlytics auto disabler." # do nothing
  else
    _PLIST_ENTRY_KEYS+=("FirebaseCrashlyticsCollectionEnabled")
    _PLIST_ENTRY_TYPES+=("bool")
    _PLIST_ENTRY_VALUES+=("NO")
  fi
else
  _PLIST_ENTRY_KEYS+=("firebase_json_raw")
  _PLIST_ENTRY_TYPES+=("string")
  _PLIST_ENTRY_VALUES+=("$_JSON_OUTPUT_BASE64")
  echo "warning:   A firebase.json file was not found, whilst this file is optional it is recommended to include it to configure firebase services in React Native Firebase."
fi;

echo "info: 2) Injecting Info.plist entries: "

# Log out the keys we're adding
for i in "${!_PLIST_ENTRY_KEYS[@]}"; do
  echo "    ->  $i) ${_PLIST_ENTRY_KEYS[$i]}" "${_PLIST_ENTRY_TYPES[$i]}" "${_PLIST_ENTRY_VALUES[$i]}"
done

for plist in "${_TARGET_PLIST}" "${_DSYM_PLIST}" ; do
  if [[ -f "${plist}" ]]; then

    # paths with spaces break the call to setPlistValue. temporarily modify
    # the shell internal field separator variable (IFS), which normally
    # includes spaces, to consist only of line breaks
    oldifs=$IFS
    IFS="
"

    for i in "${!_PLIST_ENTRY_KEYS[@]}"; do
      setPlistValue "${_PLIST_ENTRY_KEYS[$i]}" "${_PLIST_ENTRY_TYPES[$i]}" "${_PLIST_ENTRY_VALUES[$i]}" "${plist}"
    done

    # restore the original internal field separator value
    IFS=$oldifs
  else
    echo "warning:   A Info.plist build output file was not found (${plist})"
  fi
done

echo "info: <- RNFB build script finished"
```




---

## Project Files

<!-- Provide the contents of key project files which will help to debug -->
<!--     For Example: -->
<!--        - iOS: `Podfile` contents. -->
<!--        - Android: `android/build.gradle` contents. -->
<!--        - Android: `android/app/build.gradle` contents. -->
<!--        - Android: `AndroidManifest.xml` contents. -->

<!-- ADD THE CONTENTS OF THE FILES IN THE PROVIDED CODE BLOCKS BELOW -->

### Javascript

<details><summary>Click To Expand</summary>
<p>

#### `package.json`:

```json
{
  "name": "gibber",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "android": "react-native run-android",
    "ios": "react-native run-ios",
    "start": "react-native start",
    "test": "jest",
    "lint": "eslint ."
  },
  "dependencies": {
    "@react-native-community/cli": "*",
    "@react-native-community/masked-view": "^0.1.11",
    "@react-native-community/progress-bar-android": "^1.0.4",
    "@react-native-community/progress-view": "^1.3.0",
    "@react-native-firebase/app": "^12.9.2",
    "@react-native-firebase/messaging": "^12.9.2",
    "@react-navigation/material-bottom-tabs": "^5.3.15",
    "@react-navigation/native": "^5.9.4",
    "@react-navigation/stack": "^5.14.4",
    "axios": "^0.21.1",
    "react": "17.0.1",
    "react-hook-form": "^7.3.4",
    "react-native": "0.64.1",
    "react-native-eject": "^0.1.2",
    "react-native-gesture-handler": "^1.10.3",
    "react-native-paper": "^4.8.1",
    "react-native-pdf": "^6.3.0",
    "react-native-reanimated": "^2.1.0",
    "react-native-safe-area-context": "^3.2.0",
    "react-native-screens": "^3.1.1",
    "react-native-sensitive-info": "^6.0.0-alpha.9",
    "react-native-vector-icons": "^8.1.0",
    "react-native-video": "^5.1.1",
    "reanimated-bottom-sheet": "^1.0.0-alpha.22",
    "rn-fetch-blob": "^0.12.0"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9",
    "@babel/runtime": "^7.12.5",
    "@react-native-community/eslint-config": "^2.0.0",
    "babel-jest": "^26.6.3",
    "eslint": "7.14.0",
    "jest": "^26.6.3",
    "metro-react-native-babel-preset": "^0.64.0",
    "react-test-renderer": "17.0.1"
  },
  "jest": {
    "preset": "react-native"
  }
}
```

#### `firebase.json` for react-native-firebase v6:

```json
{
    "react-native": {
        
    }
}
```

</details>

### iOS

<details><summary>Click To Expand</summary>
<p>

#### `ios/Podfile`:

- [ ] I'm not using Pods
- [x] I'm using Pods and my Podfile looks like:

```ruby
require_relative '../node_modules/react-native/scripts/react_native_pods'
require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'

platform :ios, '11.0'

target 'Gibber' do
  config = use_native_modules!

  use_react_native!(
    :path => config[:reactNativePath],
    # to enable hermes on iOS, change `false` to `true` and then install pods
    :hermes_enabled => false
  )



  target 'GibberTests' do
    inherit! :complete
    # Pods for testing
  end

  # Enables Flipper.
  #
  # Note that if you have use_frameworks! enabled, Flipper will not work and
  # you should disable the next line.
  # use_flipper!('Flipper' => '0.75.1', 'Flipper-Folly' => '2.5.3', 'Flipper-RSocket' => '1.3.1')

  post_install do |installer|
    # react_native_post_install(installer)
    installer.pods_project.targets.each do |target|
      target.build_configurations.each do |config|
        #1
        deployment_target = config.build_settings['IPHONEOS_DEPLOYMENT_TARGET']
          #2
          target_components = deployment_target.split
            #3
            if target_components.length > 0
              #4
              target_initial = target_components[0].to_i
                #5
                if target_initial < 9
                  config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = "9.0"
                end
            end
            
      end
     end
  end
end
```

#### `AppDelegate.m`:

```objc
#import "AppDelegate.h"

#import <React/RCTBridge.h>
#import <React/RCTBundleURLProvider.h>
#import <React/RCTRootView.h>

#import <Firebase.h>

#ifdef FB_SONARKIT_ENABLED
#import <FlipperKit/FlipperClient.h>
#import <FlipperKitLayoutPlugin/FlipperKitLayoutPlugin.h>
#import <FlipperKitUserDefaultsPlugin/FKUserDefaultsPlugin.h>
#import <FlipperKitNetworkPlugin/FlipperKitNetworkPlugin.h>
#import <SKIOSNetworkPlugin/SKIOSNetworkAdapter.h>
#import <FlipperKitReactPlugin/FlipperKitReactPlugin.h>

static void InitializeFlipper(UIApplication *application) {
  FlipperClient *client = [FlipperClient sharedClient];
  SKDescriptorMapper *layoutDescriptorMapper = [[SKDescriptorMapper alloc] initWithDefaults];
  [client addPlugin:[[FlipperKitLayoutPlugin alloc] initWithRootNode:application withDescriptorMapper:layoutDescriptorMapper]];
  [client addPlugin:[[FKUserDefaultsPlugin alloc] initWithSuiteName:nil]];
  [client addPlugin:[FlipperKitReactPlugin new]];
  [client addPlugin:[[FlipperKitNetworkPlugin alloc] initWithNetworkAdapter:[SKIOSNetworkAdapter new]]];
  [client start];
}
#endif


@implementation AppDelegate

- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
  [FIRApp configure];
#ifdef FB_SONARKIT_ENABLED
  InitializeFlipper(application);
#endif

  RCTBridge *bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:launchOptions];
  RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge
                                                   moduleName:@"Gibber"
                                            initialProperties:nil];

  if (@available(iOS 13.0, *)) {
      rootView.backgroundColor = [UIColor systemBackgroundColor];
  } else {
      rootView.backgroundColor = [UIColor whiteColor];
  }

  self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
  UIViewController *rootViewController = [UIViewController new];
  rootViewController.view = rootView;
  self.window.rootViewController = rootViewController;
  [self.window makeKeyAndVisible];
  return YES;
}

- (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
{
#if DEBUG
  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
#else
  return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
#endif
}

@end

```

</p>
</details>

---

### Android

<details><summary>Click To Expand</summary>
<p>

#### Have you converted to AndroidX?

<!--- Mark any options that apply below -->

- [ ] my application is an AndroidX application?
- [ ] I am using `android/gradle.settings` `jetifier=true` for Android compatibility?
- [ ] I am using the NPM package `jetifier` for react-native compatibility?

#### `android/build.gradle`:

```groovy
// N/A
```

#### `android/app/build.gradle`:

```groovy
// N/A
```

#### `android/settings.gradle`:

```groovy
// N/A
```

#### `MainApplication.java`:

```java
// N/A
```

#### `AndroidManifest.xml`:

```xml
<!-- N/A -->
```

</p>
</details>

---

## Environment

<details><summary>Click To Expand</summary>
<p>

**`react-native info` output:**

<!-- Please run `react-native info` on your terminal and paste the contents into the code block below -->

```
 System:
    OS: macOS 11.6
    CPU: (8) x64 Intel(R) Core(TM) i5-1038NG7 CPU @ 2.00GHz
    Memory: 61.88 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 16.10.0 - /usr/local/bin/node
    Yarn: Not Found
    npm: 7.14.0 - ~/.npm-global/bin/npm
    Watchman: 2021.09.13.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.2 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 15.0, DriverKit 20.4, macOS 11.3, tvOS 15.0, watchOS 8.0
    Android SDK:
      API Levels: 23, 27, 28, 29, 30
      Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.1, 30.0.2
      System Images: android-28 | Google APIs Intel x86 Atom_64, android-28 | Google Play Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 2020.3 AI-203.7717.56.2031.7678000
    Xcode: 13.0/13A233 - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_291 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: * => 5.0.1 
    react: 17.0.1 => 17.0.1 
    react-native: 0.64.1 => 0.64.1 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

<!-- change `[ ]` to `[x]` to select an option(s) -->

- **Platform that you're experiencing the issue on**:
  - [x ] iOS
  - [ ] Android
  - [ ] **iOS** but have not tested behavior on Android
  - [ ] **Android** but have not tested behavior on iOS
  - [ ] Both
- **`react-native-firebase` version you're using that has this issue:**
  - `e.g. 5.4.3`
- **`Firebase` module(s) you're using that has the issue:**
  - `e.g. Instance ID`
- **Are you using `TypeScript`?**
  - `Y/N` & `VERSION`

</p>
</details>

<!-- Thanks for reading this far down ❤️  -->
<!-- High quality, detailed issues are much easier to triage for maintainers -->

<!-- For bonus points, if you put a 🔥 (:fire:) emojii at the start of the issue title we'll know -->
<!-- that you took the time to fill this out correctly, or, at least read this far -->

---

- 👉 Check out [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter for updates on the library.

> Interesting - is there any output in the build log showing what the error was? That is our Xcode script but we need to see the actual error if possible. I definitely don't see this with https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh where I/we do most of our compile testing Separately - I'd strongly recommend moving up to react-native 0.66.1 if possible - Xcode builds in general received a great deal of attention (by me personally!) between 0.64 and 0.66 ([facebook/react-native#31941](https://github.com/facebook/react-native/issues/31941) / [facebook/react-native#31480](https://github.com/facebook/react-native/issues/31480)) and I wouldn't want one of the things solved there to create problems here. https://react-native-community.github.io/upgrade-helper/ can show the way, the upgrades should be pretty easy

@mikehardy 
The terminal would technically spit out long log of the error but given that I run the program from the integrated terminal, some part of it will be cut off. I can somewhat only show you the last part of the output
```
PhaseScriptExecution [CP-User]\ [RNFB]\ Core\ Configuration /Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Script-3A3BA8C7294B359F516CA4D5.sh (in target 'Gibber' from project 'Gibber')
    cd /Users/ethan/Documents/app-development/react-native/Gibber/ios
    export ACTION\=build
    export AD_HOC_CODE_SIGNING_ALLOWED\=YES
    export ALLOW_TARGET_PLATFORM_SPECIALIZATION\=NO
    export ALTERNATE_GROUP\=staff
    export ALTERNATE_MODE\=u+w,go-w,a+rX
    export ALTERNATE_OWNER\=ethan
    export ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES\=NO
    export ALWAYS_SEARCH_USER_PATHS\=NO
    export ALWAYS_USE_SEPARATE_HEADERMAPS\=NO
    export APPLE_INTERNAL_DEVELOPER_DIR\=/AppleInternal/Developer
    export APPLE_INTERNAL_DIR\=/AppleInternal
    export APPLE_INTERNAL_DOCUMENTATION_DIR\=/AppleInternal/Documentation
    export APPLE_INTERNAL_LIBRARY_DIR\=/AppleInternal/Library
    export APPLE_INTERNAL_TOOLS\=/AppleInternal/Developer/Tools
    export APPLICATION_EXTENSION_API_ONLY\=NO
    export APPLY_RULES_IN_COPY_FILES\=NO
    export APPLY_RULES_IN_COPY_HEADERS\=NO
    export ARCHS\=x86_64
    export ARCHS_STANDARD\=arm64\ x86_64
    export ARCHS_STANDARD_32_64_BIT\=arm64\ i386\ x86_64
    export ARCHS_STANDARD_32_BIT\=i386
    export ARCHS_STANDARD_64_BIT\=arm64\ x86_64
    export ARCHS_STANDARD_INCLUDING_64_BIT\=arm64\ x86_64
    export ARCHS_UNIVERSAL_IPHONE_OS\=arm64\ i386\ x86_64
    export ASSETCATALOG_COMPILER_APPICON_NAME\=AppIcon
    export ASSETCATALOG_FILTER_FOR_DEVICE_MODEL\=iPhone13,2
    export ASSETCATALOG_FILTER_FOR_DEVICE_OS_VERSION\=15.0
    export AVAILABLE_PLATFORMS\=appletvos\ appletvsimulator\ iphoneos\ iphonesimulator\ macosx\ watchos\ watchsimulator
    export AppIdentifierPrefix\=6U9H7BH2WP.
    export BITCODE_GENERATION_MODE\=marker
    export BUILD_ACTIVE_RESOURCES_ONLY\=YES
    export BUILD_COMPONENTS\=headers\ build
    export BUILD_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products
    export BUILD_LIBRARY_FOR_DISTRIBUTION\=NO
    export BUILD_ROOT\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products
    export BUILD_STYLE\=
    export BUILD_VARIANTS\=normal
    export BUILT_PRODUCTS_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator
    export BUNDLE_CONTENTS_FOLDER_PATH_deep\=Contents/
    export BUNDLE_EXECUTABLE_FOLDER_NAME_deep\=MacOS
    export BUNDLE_FORMAT\=shallow
    export BUNDLE_FRAMEWORKS_FOLDER_PATH\=Frameworks
    export BUNDLE_PLUGINS_FOLDER_PATH\=PlugIns
    export BUNDLE_PRIVATE_HEADERS_FOLDER_PATH\=PrivateHeaders
    export BUNDLE_PUBLIC_HEADERS_FOLDER_PATH\=Headers
    export CACHE_ROOT\=/var/folders/bp/kcdmkqw934l_gmt_w4q39qdw0000gn/C/com.apple.DeveloperTools/13.0-13A233/Xcode
    export CCHROOT\=/var/folders/bp/kcdmkqw934l_gmt_w4q39qdw0000gn/C/com.apple.DeveloperTools/13.0-13A233/Xcode
    export CHMOD\=/bin/chmod
    export CHOWN\=/usr/sbin/chown
    export CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED\=YES
    export CLANG_CXX_LANGUAGE_STANDARD\=c++14
    export CLANG_CXX_LIBRARY\=libc++
    export CLANG_ENABLE_MODULES\=YES
    export CLANG_ENABLE_OBJC_ARC\=YES
    export CLANG_MODULES_BUILD_SESSION_FILE\=/Users/ethan/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/Session.modulevalidation
    export CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING\=YES
    export CLANG_WARN_BOOL_CONVERSION\=YES
    export CLANG_WARN_COMMA\=YES
    export CLANG_WARN_CONSTANT_CONVERSION\=YES
    export CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS\=YES
    export CLANG_WARN_DIRECT_OBJC_ISA_USAGE\=YES_ERROR
    export CLANG_WARN_EMPTY_BODY\=YES
    export CLANG_WARN_ENUM_CONVERSION\=YES
    export CLANG_WARN_INFINITE_RECURSION\=YES
    export CLANG_WARN_INT_CONVERSION\=YES
    export CLANG_WARN_NON_LITERAL_NULL_CONVERSION\=YES
    export CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF\=YES
    export CLANG_WARN_OBJC_LITERAL_CONVERSION\=YES
    export CLANG_WARN_OBJC_ROOT_CLASS\=YES_ERROR
    export CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER\=NO
    export CLANG_WARN_RANGE_LOOP_ANALYSIS\=YES
    export CLANG_WARN_STRICT_PROTOTYPES\=YES
    export CLANG_WARN_SUSPICIOUS_MOVE\=YES
    export CLANG_WARN_UNREACHABLE_CODE\=YES
    export CLANG_WARN__DUPLICATE_METHOD_MATCH\=YES
    export CLASS_FILE_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/JavaClasses
    export CLEAN_PRECOMPS\=YES
    export CLONE_HEADERS\=NO
    export CODESIGNING_FOLDER_PATH\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/Gibber.app
    export CODE_SIGNING_ALLOWED\=YES
    export CODE_SIGNING_REQUIRED\=YES
    export CODE_SIGN_CONTEXT_CLASS\=XCiPhoneSimulatorCodeSignContext
    export CODE_SIGN_IDENTITY\=Apple\ Development
    export CODE_SIGN_INJECT_BASE_ENTITLEMENTS\=YES
    export CODE_SIGN_STYLE\=Automatic
    export COLOR_DIAGNOSTICS\=NO
    export COMBINE_HIDPI_IMAGES\=NO
    export COMPILER_INDEX_STORE_ENABLE\=Default
    export COMPOSITE_SDK_DIRS\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/CompositeSDKs
    export COMPRESS_PNG_FILES\=YES
    export CONFIGURATION\=Debug
    export CONFIGURATION_BUILD_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator
    export CONFIGURATION_TEMP_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator
    export CONTENTS_FOLDER_PATH\=Gibber.app
    export COPYING_PRESERVES_HFS_DATA\=NO
    export COPY_HEADERS_RUN_UNIFDEF\=NO
    export COPY_PHASE_STRIP\=NO
    export COPY_RESOURCES_FROM_STATIC_FRAMEWORKS\=YES
    export CORRESPONDING_DEVICE_PLATFORM_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform
    export CORRESPONDING_DEVICE_PLATFORM_NAME\=iphoneos
    export CORRESPONDING_DEVICE_SDK_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS15.0.sdk
    export CORRESPONDING_DEVICE_SDK_NAME\=iphoneos15.0
    export CP\=/bin/cp
    export CREATE_INFOPLIST_SECTION_IN_BINARY\=NO
    export CURRENT_ARCH\=undefined_arch
    export CURRENT_PROJECT_VERSION\=1
    export CURRENT_VARIANT\=normal
    export DEAD_CODE_STRIPPING\=YES
    export DEBUGGING_SYMBOLS\=YES
    export DEBUG_INFORMATION_FORMAT\=dwarf-with-dsym
    export DEFAULT_COMPILER\=com.apple.compilers.llvm.clang.1_0
    export DEFAULT_DEXT_INSTALL_PATH\=/System/Library/DriverExtensions
    export DEFAULT_KEXT_INSTALL_PATH\=/System/Library/Extensions
    export DEFINES_MODULE\=NO
    export DEPLOYMENT_LOCATION\=NO
    export DEPLOYMENT_POSTPROCESSING\=NO
    export DEPLOYMENT_TARGET_CLANG_ENV_NAME\=IPHONEOS_DEPLOYMENT_TARGET
    export DEPLOYMENT_TARGET_CLANG_FLAG_NAME\=mios-simulator-version-min
    export DEPLOYMENT_TARGET_CLANG_FLAG_PREFIX\=-mios-simulator-version-min\=
    export DEPLOYMENT_TARGET_LD_ENV_NAME\=IPHONEOS_DEPLOYMENT_TARGET
    export DEPLOYMENT_TARGET_LD_FLAG_NAME\=ios_simulator_version_min
    export DEPLOYMENT_TARGET_SETTING_NAME\=IPHONEOS_DEPLOYMENT_TARGET
    export DEPLOYMENT_TARGET_SUGGESTED_VALUES\=9.0\ 9.1\ 9.2\ 9.3\ 10.0\ 10.1\ 10.2\ 10.3\ 11.0\ 11.1\ 11.2\ 11.3\ 11.4\ 12.0\ 12.1\ 12.2\ 12.3\ 12.4\ 13.0\ 13.1\ 13.2\ 13.3\ 13.4\ 13.5\ 13.6\ 14.0\ 14.1\ 14.2\ 14.3\ 14.4\ 14.5\ 14.6\ 14.7\ 15.0
    export DERIVED_FILES_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/DerivedSources
    export DERIVED_FILE_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/DerivedSources
    export DERIVED_SOURCES_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/DerivedSources
    export DEVELOPER_APPLICATIONS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications
    export DEVELOPER_BIN_DIR\=/Applications/Xcode.app/Contents/Developer/usr/bin
    export DEVELOPER_DIR\=/Applications/Xcode.app/Contents/Developer
    export DEVELOPER_FRAMEWORKS_DIR\=/Applications/Xcode.app/Contents/Developer/Library/Frameworks
    export DEVELOPER_FRAMEWORKS_DIR_QUOTED\=/Applications/Xcode.app/Contents/Developer/Library/Frameworks
    export DEVELOPER_LIBRARY_DIR\=/Applications/Xcode.app/Contents/Developer/Library
    export DEVELOPER_SDK_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs
    export DEVELOPER_TOOLS_DIR\=/Applications/Xcode.app/Contents/Developer/Tools
    export DEVELOPER_USR_DIR\=/Applications/Xcode.app/Contents/Developer/usr
    export DEVELOPMENT_LANGUAGE\=en
    export DEVELOPMENT_TEAM\=6U9H7BH2WP
    export DOCUMENTATION_FOLDER_PATH\=Gibber.app/en.lproj/Documentation
    export DONT_GENERATE_INFOPLIST_FILE\=NO
    export DO_HEADER_SCANNING_IN_JAM\=NO
    export DSTROOT\=/tmp/Gibber.dst
    export DT_TOOLCHAIN_DIR\=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain
    export DWARF_DSYM_FILE_NAME\=Gibber.app.dSYM
    export DWARF_DSYM_FILE_SHOULD_ACCOMPANY_PRODUCT\=NO
    export DWARF_DSYM_FOLDER_PATH\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator
    export EFFECTIVE_PLATFORM_NAME\=-iphonesimulator
    export EMBEDDED_CONTENT_CONTAINS_SWIFT\=NO
    export EMBED_ASSET_PACKS_IN_PRODUCT_BUNDLE\=NO
    export ENABLE_APP_SANDBOX\=NO
    export ENABLE_BITCODE\=NO
    export ENABLE_DEFAULT_HEADER_SEARCH_PATHS\=YES
    export ENABLE_HARDENED_RUNTIME\=NO
    export ENABLE_HEADER_DEPENDENCIES\=YES
    export ENABLE_ON_DEMAND_RESOURCES\=YES
    export ENABLE_PREVIEWS\=NO
    export ENABLE_STRICT_OBJC_MSGSEND\=YES
    export ENABLE_TESTABILITY\=YES
    export ENABLE_TESTING_SEARCH_PATHS\=NO
    export ENTITLEMENTS_DESTINATION\=__entitlements
    export ENTITLEMENTS_REQUIRED\=YES
    export EXCLUDED_ARCHS\=arm64\ 
    export EXCLUDED_INSTALLSRC_SUBDIRECTORY_PATTERNS\=.DS_Store\ .svn\ .git\ .hg\ CVS
    export EXCLUDED_RECURSIVE_SEARCH_PATH_SUBDIRECTORIES\=\*.nib\ \*.lproj\ \*.framework\ \*.gch\ \*.xcode\*\ \*.xcassets\ \(\*\)\ .DS_Store\ CVS\ .svn\ .git\ .hg\ \*.pbproj\ \*.pbxproj
    export EXECUTABLES_FOLDER_PATH\=Gibber.app/Executables
    export EXECUTABLE_FOLDER_PATH\=Gibber.app
    export EXECUTABLE_NAME\=Gibber
    export EXECUTABLE_PATH\=Gibber.app/Gibber
    export EXPANDED_CODE_SIGN_IDENTITY\=-
    export EXPANDED_CODE_SIGN_IDENTITY_NAME\=-
    export FILE_LIST\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Objects/LinkFileList
    export FIXED_FILES_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/FixedFiles
    export FRAMEWORKS_FOLDER_PATH\=Gibber.app/Frameworks
    export FRAMEWORK_FLAG_PREFIX\=-framework
    export FRAMEWORK_SEARCH_PATHS\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator\ 
    export FRAMEWORK_VERSION\=A
    export FULL_PRODUCT_NAME\=Gibber.app
    export GCC3_VERSION\=3.3
    export GCC_C_LANGUAGE_STANDARD\=gnu99
    export GCC_DYNAMIC_NO_PIC\=NO
    export GCC_INLINES_ARE_PRIVATE_EXTERN\=YES
    export GCC_NO_COMMON_BLOCKS\=YES
    export GCC_OBJC_LEGACY_DISPATCH\=YES
    export GCC_OPTIMIZATION_LEVEL\=0
    export GCC_PFE_FILE_C_DIALECTS\=c\ objective-c\ c++\ objective-c++
    export GCC_PREPROCESSOR_DEFINITIONS\=DEBUG\=1\ \ COCOAPODS\=1\ DEBUG\=1\ \ PB_FIELD_32BIT\=1\ PB_NO_PACKED_STRUCTS\=1\ PB_ENABLE_MALLOC\=1
    export GCC_SYMBOLS_PRIVATE_EXTERN\=NO
    export GCC_TREAT_WARNINGS_AS_ERRORS\=NO
    export GCC_VERSION\=com.apple.compilers.llvm.clang.1_0
    export GCC_VERSION_IDENTIFIER\=com_apple_compilers_llvm_clang_1_0
    export GCC_WARN_64_TO_32_BIT_CONVERSION\=YES
    export GCC_WARN_ABOUT_RETURN_TYPE\=YES_ERROR
    export GCC_WARN_UNDECLARED_SELECTOR\=YES
    export GCC_WARN_UNINITIALIZED_AUTOS\=YES_AGGRESSIVE
    export GCC_WARN_UNUSED_FUNCTION\=YES
    export GCC_WARN_UNUSED_VARIABLE\=YES
    export GENERATED_MODULEMAP_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/GeneratedModuleMaps-iphonesimulator
    export GENERATE_INFOPLIST_FILE\=NO
    export GENERATE_MASTER_OBJECT_FILE\=NO
    export GENERATE_PKGINFO_FILE\=YES
    export GENERATE_PROFILING_CODE\=NO
    export GENERATE_TEXT_BASED_STUBS\=NO
    export GID\=20
    export GROUP\=staff
    export HEADERMAP_INCLUDES_FLAT_ENTRIES_FOR_TARGET_BEING_BUILT\=YES
    export HEADERMAP_INCLUDES_FRAMEWORK_ENTRIES_FOR_ALL_PRODUCT_TYPES\=YES
    export HEADERMAP_INCLUDES_NONPUBLIC_NONPRIVATE_HEADERS\=YES
    export HEADERMAP_INCLUDES_PROJECT_HEADERS\=YES
    export HEADERMAP_USES_FRAMEWORK_PREFIX_ENTRIES\=YES
    export HEADERMAP_USES_VFS\=NO
    export HEADER_SEARCH_PATHS\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/include\ \ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/DoubleConversion\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/FBLazyVector\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/FBReactNativeSpec\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/Firebase\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/FirebaseCore\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/FirebaseCoreDiagnostics\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/FirebaseInstallations\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/FirebaseMessaging\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/GoogleDataTransport\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/GoogleUtilities\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/PromisesObjC\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/RCT-Folly\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/RCTRequired\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/RCTTypeSafety\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/RNCMaskedView\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/RNFBApp\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/RNFBMessaging\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/RNGestureHandler\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/RNReanimated\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/RNScreens\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/RNVectorIcons\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-Core\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-RCTBlob\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-RCTText\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-callinvoker\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-cxxreact\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-jsi\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-jsiexecutor\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-jsinspector\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-perflogger\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-runtimeexecutor\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/ReactCommon\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/Yoga\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/glog\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/nanopb\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/react-native-pdf\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/react-native-progress-view\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/react-native-safe-area-context\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/react-native-sensitive-info\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/react-native-video\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/rn-fetch-blob\"\ \ /Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Firebase/CoreOnly/Sources\ \"/Sources/FBLPromises/include\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/boost-for-react-native\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/boost-for-react-native\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/glog\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/RCT-Folly\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React-hermes\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/hermes-engine\"\ \"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Private/React-Core\"
    export HIDE_BITCODE_SYMBOLS\=YES
    export HOME\=/Users/ethan
    export ICONV\=/usr/bin/iconv
    export INFOPLIST_EXPAND_BUILD_SETTINGS\=YES
    export INFOPLIST_FILE\=Gibber/Info.plist
    export INFOPLIST_OUTPUT_FORMAT\=binary
    export INFOPLIST_PATH\=Gibber.app/Info.plist
    export INFOPLIST_PREPROCESS\=NO
    export INFOSTRINGS_PATH\=Gibber.app/en.lproj/InfoPlist.strings
    export INLINE_PRIVATE_FRAMEWORKS\=NO
    export INSTALLHDRS_COPY_PHASE\=NO
    export INSTALLHDRS_SCRIPT_PHASE\=NO
    export INSTALL_DIR\=/tmp/Gibber.dst/Applications
    export INSTALL_GROUP\=staff
    export INSTALL_MODE_FLAG\=u+w,go-w,a+rX
    export INSTALL_OWNER\=ethan
    export INSTALL_PATH\=/Applications
    export INSTALL_ROOT\=/tmp/Gibber.dst
    export IPHONEOS_DEPLOYMENT_TARGET\=11.0
    export JAVAC_DEFAULT_FLAGS\=-J-Xms64m\ -J-XX:NewSize\=4M\ -J-Dfile.encoding\=UTF8
    export JAVA_APP_STUB\=/System/Library/Frameworks/JavaVM.framework/Resources/MacOS/JavaApplicationStub
    export JAVA_ARCHIVE_CLASSES\=YES
    export JAVA_ARCHIVE_TYPE\=JAR
    export JAVA_COMPILER\=/usr/bin/javac
    export JAVA_FOLDER_PATH\=Gibber.app/Java
    export JAVA_FRAMEWORK_RESOURCES_DIRS\=Resources
    export JAVA_JAR_FLAGS\=cv
    export JAVA_SOURCE_SUBDIR\=.
    export JAVA_USE_DEPENDENCIES\=YES
    export JAVA_ZIP_FLAGS\=-urg
    export JIKES_DEFAULT_FLAGS\=+E\ +OLDCSO
    export KEEP_PRIVATE_EXTERNS\=NO
    export LD_DEPENDENCY_INFO_FILE\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Objects-normal/undefined_arch/Gibber_dependency_info.dat
    export LD_ENTITLEMENTS_SECTION\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Gibber.app-Simulated.xcent
    export LD_GENERATE_MAP_FILE\=NO
    export LD_MAP_FILE_PATH\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Gibber-LinkMap-normal-undefined_arch.txt
    export LD_NO_PIE\=NO
    export LD_QUOTE_LINKER_ARGUMENTS_FOR_COMPILER_DRIVER\=YES
    export LD_RUNPATH_SEARCH_PATHS\=/usr/lib/swift\ \ @executable_path/Frameworks
    export LEGACY_DEVELOPER_DIR\=/Applications/Xcode.app/Contents/PlugIns/Xcode3Core.ideplugin/Contents/SharedSupport/Developer
    export LEX\=lex
    export LIBRARY_DEXT_INSTALL_PATH\=/Library/DriverExtensions
    export LIBRARY_FLAG_NOSPACE\=YES
    export LIBRARY_FLAG_PREFIX\=-l
    export LIBRARY_KEXT_INSTALL_PATH\=/Library/Extensions
    export LIBRARY_SEARCH_PATHS\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator\ \"/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator\"\ \"/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift-5.0/iphonesimulator\"\ \"\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/DoubleConversion\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/FBReactNativeSpec\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/FirebaseCore\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/FirebaseCoreDiagnostics\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/FirebaseInstallations\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/FirebaseMessaging\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/GoogleDataTransport\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/GoogleUtilities\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/PromisesObjC\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/RCT-Folly\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/RCTTypeSafety\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/RNCMaskedView\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/RNFBApp\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/RNFBMessaging\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/RNGestureHandler\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/RNReanimated\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/RNScreens\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/RNVectorIcons\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-Core\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-CoreModules\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-RCTAnimation\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-RCTBlob\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-RCTImage\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-RCTLinking\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-RCTNetwork\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-RCTSettings\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-RCTText\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-RCTVibration\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-cxxreact\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-jsi\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-jsiexecutor\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-jsinspector\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/React-perflogger\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/ReactCommon\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/Yoga\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/glog\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/nanopb\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/react-native-pdf\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/react-native-progress-view\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/react-native-safe-area-context\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/react-native-sensitive-info\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/react-native-video\"\ \"/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/rn-fetch-blob\"
    export LINKER_DISPLAYS_MANGLED_NAMES\=NO
    export LINK_FILE_LIST_normal_x86_64\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Objects-normal/x86_64/Gibber.LinkFileList
    export LINK_WITH_STANDARD_LIBRARIES\=YES
    export LLVM_TARGET_TRIPLE_OS_VERSION\=ios11.0
    export LLVM_TARGET_TRIPLE_SUFFIX\=-simulator
    export LLVM_TARGET_TRIPLE_VENDOR\=apple
    export LOCALIZATION_EXPORT_SUPPORTED\=YES
    export LOCALIZED_RESOURCES_FOLDER_PATH\=Gibber.app/en.lproj
    export LOCALIZED_STRING_MACRO_NAMES\=NSLocalizedString\ CFCopyLocalizedString
    export LOCALIZED_STRING_SWIFTUI_SUPPORT\=YES
    export LOCAL_ADMIN_APPS_DIR\=/Applications/Utilities
    export LOCAL_APPS_DIR\=/Applications
    export LOCAL_DEVELOPER_DIR\=/Library/Developer
    export LOCAL_LIBRARY_DIR\=/Library
    export LOCROOT\=/Users/ethan/Documents/app-development/react-native/Gibber/ios
    export LOCSYMROOT\=/Users/ethan/Documents/app-development/react-native/Gibber/ios
    export MACH_O_TYPE\=mh_execute
    export MAC_OS_X_PRODUCT_BUILD_VERSION\=20G165
    export MAC_OS_X_VERSION_ACTUAL\=110600
    export MAC_OS_X_VERSION_MAJOR\=110000
    export MAC_OS_X_VERSION_MINOR\=110600
    export METAL_LIBRARY_FILE_BASE\=default
    export METAL_LIBRARY_OUTPUT_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/Gibber.app
    export MODULES_FOLDER_PATH\=Gibber.app/Modules
    export MODULE_CACHE_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/ModuleCache.noindex
    export MTL_ENABLE_DEBUG_INFO\=YES
    export NATIVE_ARCH\=x86_64
    export NATIVE_ARCH_32_BIT\=i386
    export NATIVE_ARCH_64_BIT\=x86_64
    export NATIVE_ARCH_ACTUAL\=x86_64
    export NO_COMMON\=YES
    export OBJC_ABI_VERSION\=2
    export OBJECT_FILE_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Objects
    export OBJECT_FILE_DIR_normal\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Objects-normal
    export OBJROOT\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex
    export ONLY_ACTIVE_ARCH\=YES
    export OS\=MACOS
    export OSAC\=/usr/bin/osacompile
    export OTHER_CFLAGS\=\ -fmodule-map-file\=\"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React/React-Core.modulemap\"\ -fmodule-map-file\=\"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/yoga/Yoga.modulemap\"\ \ -DFOLLY_NO_CONFIG\ -DFOLLY_MOBILE\=1\ -DFOLLY_USE_LIBCPP\=1\ -DRNVERSION\=64
    export OTHER_CPLUSPLUSFLAGS\=\ -fmodule-map-file\=\"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React/React-Core.modulemap\"\ -fmodule-map-file\=\"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/yoga/Yoga.modulemap\"\ \ -DFOLLY_NO_CONFIG\ -DFOLLY_MOBILE\=1\ -DFOLLY_USE_LIBCPP\=1\ -DRNVERSION\=64
    export OTHER_LDFLAGS\=\ -ObjC\ -l\"DoubleConversion\"\ -l\"FBReactNativeSpec\"\ -l\"FirebaseCore\"\ -l\"FirebaseCoreDiagnostics\"\ -l\"FirebaseInstallations\"\ -l\"FirebaseMessaging\"\ -l\"GoogleDataTransport\"\ -l\"GoogleUtilities\"\ -l\"PromisesObjC\"\ -l\"RCT-Folly\"\ -l\"RCTTypeSafety\"\ -l\"RNCMaskedView\"\ -l\"RNFBApp\"\ -l\"RNFBMessaging\"\ -l\"RNGestureHandler\"\ -l\"RNReanimated\"\ -l\"RNScreens\"\ -l\"RNVectorIcons\"\ -l\"React-Core\"\ -l\"React-CoreModules\"\ -l\"React-RCTAnimation\"\ -l\"React-RCTBlob\"\ -l\"React-RCTImage\"\ -l\"React-RCTLinking\"\ -l\"React-RCTNetwork\"\ -l\"React-RCTSettings\"\ -l\"React-RCTText\"\ -l\"React-RCTVibration\"\ -l\"React-cxxreact\"\ -l\"React-jsi\"\ -l\"React-jsiexecutor\"\ -l\"React-jsinspector\"\ -l\"React-perflogger\"\ -l\"ReactCommon\"\ -l\"Yoga\"\ -l\"glog\"\ -l\"nanopb\"\ -l\"react-native-pdf\"\ -l\"react-native-progress-view\"\ -l\"react-native-safe-area-context\"\ -l\"react-native-sensitive-info\"\ -l\"react-native-video\"\ -l\"rn-fetch-blob\"\ -l\"sqlite3\"\ -l\"stdc++\"\ -l\"z\"\ -framework\ \"AudioToolbox\"\ -framework\ \"CoreTelephony\"\ -framework\ \"Foundation\"\ -framework\ \"JavaScriptCore\"\ -framework\ \"MobileCoreServices\"\ -framework\ \"Security\"\ -framework\ \"SystemConfiguration\"\ -framework\ \"UIKit\"\ -weak_framework\ \"UserNotifications\"\ -ObjC\ -lc++
    export OTHER_SWIFT_FLAGS\=\ -D\ COCOAPODS\ -Xcc\ -fmodule-map-file\=\"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/React/React-Core.modulemap\"\ -Xcc\ -fmodule-map-file\=\"/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods/Headers/Public/yoga/Yoga.modulemap\"
    export PACKAGE_TYPE\=com.apple.package-type.wrapper.application
    export PASCAL_STRINGS\=YES
    export PATH\=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin:/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/local/bin:/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/libexec:/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/usr/bin:/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/usr/local/bin:/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/local/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/local/bin:/Users/ethan/Documents/app-development/react-native/Gibber/node_modules/.bin:/Users/ethan/Documents/app-development/react-native/node_modules/.bin:/Users/ethan/Documents/app-development/node_modules/.bin:/Users/ethan/Documents/node_modules/.bin:/Users/ethan/node_modules/.bin:/Users/node_modules/.bin:/node_modules/.bin:/Users/ethan/.npm-global/lib/node_modules/npm/node_modules/@npmcli/run-script/lib/node-gyp-bin:/Applications/XAMPP/xamppfiles/bin:/Users/ethan/.pyenv/shims:/Users/ethan/.pyenv/bin:/Users/ethan/.npm-global/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware\ Fusion.app/Contents/Public:/Library/Apple/usr/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/ethan/Flutter/flutter/bin:/Users/ethan/Library/Android/sdk/platform-tools:/Users/ethan/Library/Android/sdk/tools:/Users/ethan/Library/Android/sdk/tools/bin:/Users/ethan/Library/Android/sdk/emulatorexport
    export PATH_PREFIXES_EXCLUDED_FROM_HEADER_DEPENDENCIES\=/usr/include\ /usr/local/include\ /System/Library/Frameworks\ /System/Library/PrivateFrameworks\ /Applications/Xcode.app/Contents/Developer/Headers\ /Applications/Xcode.app/Contents/Developer/SDKs\ /Applications/Xcode.app/Contents/Developer/Platforms
    export PBDEVELOPMENTPLIST_PATH\=Gibber.app/pbdevelopment.plist
    export PER_ARCH_OBJECT_FILE_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Objects-normal/undefined_arch
    export PER_VARIANT_OBJECT_FILE_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Objects-normal
    export PKGINFO_FILE_PATH\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/PkgInfo
    export PKGINFO_PATH\=Gibber.app/PkgInfo
    export PLATFORM_DEVELOPER_APPLICATIONS_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Applications
    export PLATFORM_DEVELOPER_BIN_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin
    export PLATFORM_DEVELOPER_LIBRARY_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library
    export PLATFORM_DEVELOPER_SDK_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs
    export PLATFORM_DEVELOPER_TOOLS_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Tools
    export PLATFORM_DEVELOPER_USR_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr
    export PLATFORM_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform
    export PLATFORM_DISPLAY_NAME\=iOS\ Simulator
    export PLATFORM_FAMILY_NAME\=iOS
    export PLATFORM_NAME\=iphonesimulator
    export PLATFORM_PREFERRED_ARCH\=x86_64
    export PLATFORM_PRODUCT_BUILD_VERSION\=19A339
    export PLIST_FILE_OUTPUT_FORMAT\=binary
    export PLUGINS_FOLDER_PATH\=Gibber.app/PlugIns
    export PODS_BUILD_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products
    export PODS_CONFIGURATION_BUILD_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator
    export PODS_PODFILE_DIR_PATH\=/Users/ethan/Documents/app-development/react-native/Gibber/ios/.
    export PODS_ROOT\=/Users/ethan/Documents/app-development/react-native/Gibber/ios/Pods
    export PODS_XCFRAMEWORKS_BUILD_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/XCFrameworkIntermediates
    export PRECOMPS_INCLUDE_HEADERS_FROM_BUILT_PRODUCTS_DIR\=YES
    export PRECOMP_DESTINATION_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/PrefixHeaders
    export PRESERVE_DEAD_CODE_INITS_AND_TERMS\=NO
    export PRIVATE_HEADERS_FOLDER_PATH\=Gibber.app/PrivateHeaders
    export PRODUCT_BUNDLE_IDENTIFIER\=com.simsview.gibber
    export PRODUCT_BUNDLE_PACKAGE_TYPE\=APPL
    export PRODUCT_MODULE_NAME\=Gibber
    export PRODUCT_NAME\=Gibber
    export PRODUCT_SETTINGS_PATH\=/Users/ethan/Documents/app-development/react-native/Gibber/ios/Gibber/Info.plist
    export PRODUCT_TYPE\=com.apple.product-type.application
    export PROFILING_CODE\=NO
    export PROJECT\=Gibber
    export PROJECT_DERIVED_FILE_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/DerivedSources
    export PROJECT_DIR\=/Users/ethan/Documents/app-development/react-native/Gibber/ios
    export PROJECT_FILE_PATH\=/Users/ethan/Documents/app-development/react-native/Gibber/ios/Gibber.xcodeproj
    export PROJECT_NAME\=Gibber
    export PROJECT_TEMP_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build
    export PROJECT_TEMP_ROOT\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex
    export PUBLIC_HEADERS_FOLDER_PATH\=Gibber.app/Headers
    export RECURSIVE_SEARCH_PATHS_FOLLOW_SYMLINKS\=YES
    export REMOVE_CVS_FROM_RESOURCES\=YES
    export REMOVE_GIT_FROM_RESOURCES\=YES
    export REMOVE_HEADERS_FROM_EMBEDDED_BUNDLES\=YES
    export REMOVE_HG_FROM_RESOURCES\=YES
    export REMOVE_SVN_FROM_RESOURCES\=YES
    export REZ_COLLECTOR_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/ResourceManagerResources
    export REZ_OBJECTS_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/ResourceManagerResources/Objects
    export REZ_SEARCH_PATHS\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator\ 
    export SCAN_ALL_SOURCE_FILES_FOR_INCLUDES\=NO
    export SCRIPTS_FOLDER_PATH\=Gibber.app/Scripts
    export SCRIPT_INPUT_FILE_0\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/Gibber.app/Info.plist
    export SCRIPT_INPUT_FILE_COUNT\=1
    export SCRIPT_INPUT_FILE_LIST_COUNT\=0
    export SCRIPT_OUTPUT_FILE_COUNT\=0
    export SCRIPT_OUTPUT_FILE_LIST_COUNT\=0
    export SDKROOT\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk
    export SDK_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk
    export SDK_DIR_iphonesimulator\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk
    export SDK_DIR_iphonesimulator15_0\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk
    export SDK_NAME\=iphonesimulator15.0
    export SDK_NAMES\=iphonesimulator15.0
    export SDK_PRODUCT_BUILD_VERSION\=19A339
    export SDK_VERSION\=15.0
    export SDK_VERSION_ACTUAL\=150000
    export SDK_VERSION_MAJOR\=150000
    export SDK_VERSION_MINOR\=150000
    export SED\=/usr/bin/sed
    export SEPARATE_STRIP\=NO
    export SEPARATE_SYMBOL_EDIT\=NO
    export SET_DIR_MODE_OWNER_GROUP\=YES
    export SET_FILE_MODE_OWNER_GROUP\=NO
    export SHALLOW_BUNDLE\=YES
    export SHARED_DERIVED_FILE_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator/DerivedSources
    export SHARED_FRAMEWORKS_FOLDER_PATH\=Gibber.app/SharedFrameworks
    export SHARED_PRECOMPS_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/PrecompiledHeaders
    export SHARED_SUPPORT_FOLDER_PATH\=Gibber.app/SharedSupport
    export SKIP_INSTALL\=NO
    export SOURCE_ROOT\=/Users/ethan/Documents/app-development/react-native/Gibber/ios
    export SRCROOT\=/Users/ethan/Documents/app-development/react-native/Gibber/ios
    export STRINGSDATA_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Objects-normal/undefined_arch
    export STRINGSDATA_ROOT\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build
    export STRINGS_FILE_INFOPLIST_RENAME\=YES
    export STRINGS_FILE_OUTPUT_ENCODING\=binary
    export STRIP_BITCODE_FROM_COPIED_FILES\=NO
    export STRIP_INSTALLED_PRODUCT\=NO
    export STRIP_STYLE\=all
    export STRIP_SWIFT_SYMBOLS\=YES
    export SUPPORTED_DEVICE_FAMILIES\=1,2
    export SUPPORTED_PLATFORMS\=iphoneos\ iphonesimulator
    export SUPPORTS_TEXT_BASED_API\=NO
    export SWIFT_EMIT_LOC_STRINGS\=NO
    export SWIFT_OPTIMIZATION_LEVEL\=-Onone
    export SWIFT_PLATFORM_TARGET_PREFIX\=ios
    export SWIFT_RESPONSE_FILE_PATH_normal_x86_64\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Objects-normal/x86_64/Gibber.SwiftFileList
    export SWIFT_VERSION\=5.0
    export SYMROOT\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products
    export SYSTEM_ADMIN_APPS_DIR\=/Applications/Utilities
    export SYSTEM_APPS_DIR\=/Applications
    export SYSTEM_CORE_SERVICES_DIR\=/System/Library/CoreServices
    export SYSTEM_DEMOS_DIR\=/Applications/Extras
    export SYSTEM_DEVELOPER_APPS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications
    export SYSTEM_DEVELOPER_BIN_DIR\=/Applications/Xcode.app/Contents/Developer/usr/bin
    export SYSTEM_DEVELOPER_DEMOS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications/Utilities/Built\ Examples
    export SYSTEM_DEVELOPER_DIR\=/Applications/Xcode.app/Contents/Developer
    export SYSTEM_DEVELOPER_DOC_DIR\=/Applications/Xcode.app/Contents/Developer/ADC\ Reference\ Library
    export SYSTEM_DEVELOPER_GRAPHICS_TOOLS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications/Graphics\ Tools
    export SYSTEM_DEVELOPER_JAVA_TOOLS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications/Java\ Tools
    export SYSTEM_DEVELOPER_PERFORMANCE_TOOLS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications/Performance\ Tools
    export SYSTEM_DEVELOPER_RELEASENOTES_DIR\=/Applications/Xcode.app/Contents/Developer/ADC\ Reference\ Library/releasenotes
    export SYSTEM_DEVELOPER_TOOLS\=/Applications/Xcode.app/Contents/Developer/Tools
    export SYSTEM_DEVELOPER_TOOLS_DOC_DIR\=/Applications/Xcode.app/Contents/Developer/ADC\ Reference\ Library/documentation/DeveloperTools
    export SYSTEM_DEVELOPER_TOOLS_RELEASENOTES_DIR\=/Applications/Xcode.app/Contents/Developer/ADC\ Reference\ Library/releasenotes/DeveloperTools
    export SYSTEM_DEVELOPER_USR_DIR\=/Applications/Xcode.app/Contents/Developer/usr
    export SYSTEM_DEVELOPER_UTILITIES_DIR\=/Applications/Xcode.app/Contents/Developer/Applications/Utilities
    export SYSTEM_DEXT_INSTALL_PATH\=/System/Library/DriverExtensions
    export SYSTEM_DOCUMENTATION_DIR\=/Library/Documentation
    export SYSTEM_KEXT_INSTALL_PATH\=/System/Library/Extensions
    export SYSTEM_LIBRARY_DIR\=/System/Library
    export TAPI_VERIFY_MODE\=ErrorsOnly
    export TARGETED_DEVICE_FAMILY\=1
    export TARGETNAME\=Gibber
    export TARGET_BUILD_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Products/Debug-iphonesimulator
    export TARGET_DEVICE_IDENTIFIER\=963EB83A-66CB-4790-93E9-E16C2A9006E3
    export TARGET_DEVICE_MODEL\=iPhone13,2
    export TARGET_DEVICE_OS_VERSION\=15.0
    export TARGET_DEVICE_PLATFORM_NAME\=iphonesimulator
    export TARGET_NAME\=Gibber
    export TARGET_TEMP_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build
    export TEMP_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build
    export TEMP_FILES_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build
    export TEMP_FILE_DIR\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build
    export TEMP_ROOT\=/Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex
    export TEST_FRAMEWORK_SEARCH_PATHS\=\ /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks\ /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk/Developer/Library/Frameworks
    export TEST_LIBRARY_SEARCH_PATHS\=\ /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/lib
    export TOOLCHAINS\=com.apple.dt.toolchain.XcodeDefault
    export TOOLCHAIN_DIR\=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain
    export TREAT_MISSING_BASELINES_AS_TEST_FAILURES\=NO
    export TeamIdentifierPrefix\=6U9H7BH2WP.
    export UID\=501
    export UNLOCALIZED_RESOURCES_FOLDER_PATH\=Gibber.app
    export UNSTRIPPED_PRODUCT\=NO
    export USER\=ethan
    export USER_APPS_DIR\=/Users/ethan/Applications
    export USER_LIBRARY_DIR\=/Users/ethan/Library
    export USE_DYNAMIC_NO_PIC\=YES
    export USE_HEADERMAP\=YES
    export USE_HEADER_SYMLINKS\=NO
    export USE_LLVM_TARGET_TRIPLES\=YES
    export USE_LLVM_TARGET_TRIPLES_FOR_CLANG\=YES
    export USE_LLVM_TARGET_TRIPLES_FOR_LD\=YES
    export USE_LLVM_TARGET_TRIPLES_FOR_TAPI\=YES
    export USE_RECURSIVE_SCRIPT_INPUTS_IN_SCRIPT_PHASES\=YES
    export VALIDATE_DEVELOPMENT_ASSET_PATHS\=YES_ERROR
    export VALIDATE_PRODUCT\=NO
    export VALIDATE_WORKSPACE\=YES_ERROR
    export VALID_ARCHS\=arm64\ arm64e\ i386\ x86_64
    export VERBOSE_PBXCP\=NO
    export VERSIONING_SYSTEM\=apple-generic
    export VERSIONPLIST_PATH\=Gibber.app/version.plist
    export VERSION_INFO_BUILDER\=ethan
    export VERSION_INFO_FILE\=Gibber_vers.c
    export VERSION_INFO_STRING\=\"@\(\#\)PROGRAM:Gibber\ \ PROJECT:Gibber-1\"
    export WRAPPER_EXTENSION\=app
    export WRAPPER_NAME\=Gibber.app
    export WRAPPER_SUFFIX\=.app
    export WRAP_ASSET_PACKS_IN_SEPARATE_DIRECTORIES\=NO
    export XCODE_APP_SUPPORT_DIR\=/Applications/Xcode.app/Contents/Developer/Library/Xcode
    export XCODE_PRODUCT_BUILD_VERSION\=13A233
    export XCODE_VERSION_ACTUAL\=1300
    export XCODE_VERSION_MAJOR\=1300
    export XCODE_VERSION_MINOR\=1300
    export XPCSERVICES_FOLDER_PATH\=Gibber.app/XPCServices
    export YACC\=yacc
    export arch\=undefined_arch
    export variant\=normal
    /bin/sh -c /Users/ethan/Library/Developer/Xcode/DerivedData/Gibber-bqmcphduhmtknhflnzqxcjarabsv/Build/Intermediates.noindex/Gibber.build/Debug-iphonesimulator/Gibber.build/Script-3A3BA8C7294B359F516CA4D5.sh
info: -> RNFB build script started
info: 1) Locating firebase.json file:
info:      (1 of 2) Searching in '/Users/ethan/Documents/app-development/react-native/Gibber' for a firebase.json file.
Unknown option: n
Usage: head [-options] <url>...
    -m <method>   use method for the request (default is 'HEAD')
    -f            make request even if head believes method is illegal
    -b <base>     Use the specified URL as base
    -t <timeout>  Set timeout value
    -i <time>     Set the If-Modified-Since header on the request
    -c <conttype> use this content-type for POST, PUT, CHECKIN
    -a            Use text mode for content I/O
    -p <proxyurl> use this as a proxy
    -P            don't load proxy settings from environment
    -H <header>   send this HTTP header (you can specify several)

    -u            Display method and URL before any response
    -U            Display request headers (implies -u)
    -s            Display response status code
    -S            Display response status chain
    -e            Display response headers
    -d            Do not display content
    -o <format>   Process HTML content in various ways

    -v            Show program version
    -h            Print this message

    -x            Extra debugging output
Command PhaseScriptExecution failed with a nonzero exit code


** BUILD FAILED **
```
 
Also, with regards to moving up to react-native 0.66.1, I made a standalone test app to run react native firebase with the same exact configurations and it still throws the exact same error.

The funny part is that I just tried building this on xcode directly and it **works**. I know that if I have no choice the only workaround is for me to just run my app each time from xcode

--- a/packages/app/ios_config.sh
+++ b/packages/app/ios_config.sh
@@ -66,7 +66,7 @@ while true; do
   _CURRENT_SEARCH_DIR=$(dirname "$_CURRENT_SEARCH_DIR")
   if [[ "$_CURRENT_SEARCH_DIR" == "/" ]] || [[ ${_CURRENT_LOOKUPS} -gt ${_MAX_LOOKUPS} ]]; then break; fi;
   echo "info:      ($_CURRENT_LOOKUPS of $_MAX_LOOKUPS) Searching in '$_CURRENT_SEARCH_DIR' for a ${_JSON_FILE_NAME} file."
-  _SEARCH_RESULT=$(find "$_CURRENT_SEARCH_DIR" -maxdepth 2 -name ${_JSON_FILE_NAME} -print | head -n 1)
+  _SEARCH_RESULT=$(find "$_CURRENT_SEARCH_DIR" -maxdepth 2 -name ${_JSON_FILE_NAME} -print | /usr/bin/head -n 1)
   if [[ ${_SEARCH_RESULT} ]]; then
     echo "info:      ${_JSON_FILE_NAME} found at $_SEARCH_RESULT"
     break;

